x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_requirements_page/fleet_server_requirement_page.tsx

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLoadingSpinner,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiLink,
+  EuiEmptyPrompt,
+} from '@elastic/eui';
+import styled from 'styled-components';
+import { FormattedMessage } from 'react-intl';
+
+import { useStartServices } from '../../../hooks';
+
+export const ContentWrapper = styled(EuiFlexGroup)`
+  height: 100%;
+`;
+
+function renderOnPremInstructions() {
+  return (
+    <EuiPanel
+      grow={false}
+      hasShadow={false}
+      hasBorder={true}
+      className="eui-textCenter"
+      paddingSize="l"
```

Comment:
change `paddingSize` to `"none"`. EuiEmptryPrompt comes with its own padding, so the extra padding from the panel is not necessary. My designs use 24px of padding between the empty prompt content and panel edge.